### DISCUSSION TITLE
Integration with arize-phoenix

### DISCUSSION THREAD

**Discussion by v-johnsen** [run-llama/llama_deploy#128] [Q&A]
Is there any integration for arize-phoenix with llama-agents? I have tried the llama_index.core.set_global_handler("arize_phoenix"), but it dosnt seem to work with llama-agents(?)

**Comment by altonalexander** [comment]
There is a really good example and documentation here: https://docs.arize.com/phoenix/tracing/integrations-tracing/llamaindex
